Button/Button.module.less

```diff
@@ -5,178 +5,210 @@
 @import "../styles/skin.less";
 
 .button {
-	.applySkins({
-		height: @agate-button-height;
-		font-weight: @agate-button-font-weight;
-		letter-spacing: @agate-button-letter-spacing;
-		text-indent: @agate-button-text-indent;  // Keep the text centered
-		font-variant-ligatures: @agate-button-font-ligatures;
-		min-width: @agate-button-height;
-		padding: 0 @agate-button-h-padding;
-		margin: 0 @agate-component-spacing;
-		box-sizing: border-box;
-		font-size: @agate-button-font-size;
-		vertical-align: middle;
-
-		&.minWidth {
-			padding: 0;
-
-			.client .icon {
-				margin: 0;
+	.applySkins( {
```

Comment:
Can you remove this extra space?  Also, the indentation below seems one level too many.